optimize/backend/src/it/java/io/camunda/optimize/service/importing/ZeebeUserTaskImportIT.java

```diff
@@ -938,6 +976,14 @@ private long getDurationInMsBetweenStartAndFirstAssignOperation(
         .toMillis();
   }
 
+  private long getDurationInMsBetweenStartAndLastAssignOperation(
```

Comment:
```suggestion
  private long getDurationInMsBetweenStartDateAndLastAssignedOperation(
```
Just to clarify that the duration is not between different assigned events, but between the actual start date and the last assigned event